.travis.yml

```diff
@@ -34,12 +42,7 @@ stages:
 
 # These steps are taken from: http://cassandra.apache.org/doc/latest/getting_started/installing.html#installation-from-binary-tarball-files.
 before_install:
-  - java -version
-  - wget https://archive.apache.org/dist/cassandra/$CASSANDRA_VERSION/apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz
-  - tar -xzf apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz
-  - |-
-    if [ $AUTHENTICATION = true ]; then
-      sed -i -e "s/\(authenticator: \)AllowAllAuthenticator/\1PasswordAuthenticator/" apache-cassandra-$CASSANDRA_VERSION/conf/cassandra.yaml
-    fi
-  - sh apache-cassandra-$CASSANDRA_VERSION/bin/cassandra
-  - for try in {1..12}; do (nodetool version > /dev/null 2>&1 && break) || sleep 5; done
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
```

Comment:
Do we really need to move it to `/usr/local/bin`? Guess it is fine to run it from where it loaded to.